src/index.ts

```diff
@@ -80,18 +87,28 @@ async function generateSvgSprite({
   outputPath,
   outputDirRelative,
   iconNameTransformer,
+  svgo,
 }: {
   files: string[];
   inputDir: string;
   outputPath: string;
   outputDirRelative?: string;
   iconNameTransformer?: (fileName: string) => string;
+  svgo?: {
+    enabled?: boolean;
+    options?: SvgoConfig;
+  };
```

Comment:
Suggestion for an alternative API:

```suggestion
  svgo?: boolean | SvgoConfig
```

Reply:
Thanks, that's a pretty good idea